lms/djangoapps/mobile_api/testutils.py

```diff
@@ -204,3 +223,183 @@ def test_unenrolled_user(self):
         self.unenroll()
         response = self.api_response(expected_response_code=None)
         self.verify_failure(response)
+
+
+class TestMobileAPIMilestones(MobileAPITestCase, MobileCourseAccessTestMixin):
```

Comment:
The classes in this file are intended to be Mixins or Test Base classes to be used by other Tests.  
Since all APIs that are decorated with `mobile_course_access` need to also make the milestone check, all their test classes should also run these tests.